src/formats/notion/convert-to-md.ts

```diff
@@ -283,16 +315,51 @@ function fixNotionDates(body: HTMLElement) {
 	});
 }
 
-function fixIndented(body: HTMLElement) {
-	body.findAll('div.indented').forEach(div => div.remove());
+function fixToggleHeadings(body: HTMLElement) {
+	const toggleHeadings = body.findAll('summary')
+	toggleHeadings.forEach((heading) => {
+		const fontSizeToHeadings: Record<string, 'h1' | 'h2' | 'h3'> = {
+			'1.875em': 'h1',
+			'1.5em': 'h2',
+			'1.25em': 'h3',
+		}
+		const style = heading.getAttribute('style')
+		if (!style) {
+			return
+		} else {
+			for (let key of Object.keys(fontSizeToHeadings)) {
+				if (style.includes(key)) {
+					const newHeading = document.createElement(fontSizeToHeadings[key])
+					newHeading.setText(heading.textContent ?? '')
+					heading.replaceWith(newHeading)
+					return
+				}
+			}
+		}
+		
+	})
+}
+
+function replaceElementsWithChildren(body: HTMLElement, selector: string) {
+	let element = body.querySelector(selector)
```

Comment:
Why not use findAll?

Reply:
Searching for the first element in a `while` loop ensured that the element hasn't been removed in a previous iteration (i.e. nested children etc). Was having trouble with the `findAll` function

Reply:
Hmmm it seems that we are only removing the exact element being found while all nested children are reattached to the parent, so there shouldn't be any elements that gets removed from a previous iteration, no?

Reply:
I can test using `findAll`.